app/controllers/admin/api/signups_controller.rb

```diff
@@ -25,8 +25,8 @@ def check_creation_errors
     # and not respond with pathetic error
     raise ActiveRecord::RecordNotFound if @signup_result.errors[:plans].present?
 
-    @signup_result.user.errors.each do |attr, error|
-      @signup_result.account.errors.add(attr, error)
+    @signup_result.user.errors.each do |error|
```

Comment:
```
DEPRECATION WARNING: Enumerating ActiveModel::Errors as a hash has been deprecated.
In Rails 6.1, `errors` is an array of Error objects,
therefore it should be accessed by a block with a single block
parameter like this:

person.errors.each do |error|
  attribute = error.attribute
  message = error.message
end

You are passing a block expecting two parameters,
so the old hash behavior is simulated. As this is deprecated,
this will result in an ArgumentError in Rails 7.0.
```